src/orders/components/OrderReturnPage/OrderReturnRefundItemsCard/ReturnItemsCard.tsx

```diff
@@ -192,10 +177,20 @@ const ItemsCard: React.FC<OrderReturnRefundLinesCardProps> = ({
                   <TableCellAvatar
                     thumbnail={thumbnail?.url}
                     style={{ width: productNameCellWidth }}
+                    badge={
+                      isDeleted && (
+                        <StatusBadge
+                          variant="error"
+                          description={intl.formatMessage(
+                            messages.deletedVariant
+                          )}
+                        />
+                      )
+                    }
                   >
                     {productName || <Skeleton />}
                   </TableCellAvatar>
-                  <ProductErrorCell hasVariant={isReturnable} />
+                  <TableCell />
```

Comment:
Kind of. I added an empty TableCell here because second row shifted to the left. Another solution is to just remove additional TableCell from the header row of the table. However, the UI ends up slightly different, because cells are now longer.

![image](https://user-images.githubusercontent.com/41952692/134910182-7e72b54d-490d-43e1-9f43-410762e3d28d.png)

Adding extra TableCell here is somewhat a shortcut, so I think it is necessary to adjust the styling to better match the designs. I'll look into it.